tests/deprecation/test_use_l10n.py

```diff
@@ -0,0 +1,41 @@
+import sys
+from types import ModuleType
+
+from django.conf import USE_L10N_DEPRECATED_MSG, Settings, settings
+from django.test import TestCase, ignore_warnings
+from django.utils.deprecation import RemovedInDjango50Warning
+
+
+class DeprecationTests(TestCase):
+    msg = USE_L10N_DEPRECATED_MSG
+
+    def test_override_settings_warning(self):
+        with self.assertRaisesMessage(RemovedInDjango50Warning, self.msg):
+            with self.settings(USE_L10N=True):
+                pass
```

Comment:
This looks at first glance like it's doing "_nothing_" but I think it's actually the test responsible for checking `UserSettingsHolder.__setattr__` raises, right? (ie: what will happen if you use `django.conf.settings.configure(...)`) it just happens that `override_settings` uses a `UserSettingsHolder` under the hood, so to speak?

Might be worth a comment to indicate what's actually happening? (If I'm right, well I've guessed correctly. If I'm wrong that'd be a good argument for the comment ;))

Reply:
I added a comment.